boofuzz/primitives/random_data.py

```diff
@@ -72,7 +72,7 @@ def mutate(self):
             length = self.min_length + self._mutant_index * self.step
 
         # reset the value and generate a random string of the determined length.
-        self._value = b""
+        self._value = six.binary_type(b"")
```

Comment:
Actually, using `b""` is enough.

Reply:
True, but for consistency and explicitness is why I went with using the binary_type